docs/incident-management.md

```diff
@@ -0,0 +1,123 @@
+# Incident Management for InstructLab
+
+## Attribution
+Before we start, I would like to mention that this document and process is inspired
+by and influenced heavily from [here][gist] originally authored by [coderanger][ranger].
+
+## Problem Statement
+
+With the multiple systems that come together for the open source project InstructLab
+to work, we need a catalogue of major issues and problems so we can track and find
+critical bugs in the open. Bulding a formalized incident management process is
+the first step in building this knowledge database.
+
+## Proposed Solution
+
+Create a light weight but formalized Incident Management process. Whomever calls
+for the Incident is the initial "Incident Commander" [IC] and becomes the hub of
+the information, starts the conversation in the shared Slack channel
+and has a shared timeline (google doc or the like).
+After "a reasonable" amount of time they can hand off the IC
+to someone else, and so on till the Incident is resolved.
+
+### What counts as an incident?
+
+Anything with customer or community visible negative consequences. In most cases this
```

Comment:
Customer? This sounds a bit strange given this is an open source project. "Community" seems sufficient.

Reply:
agreed

Reply:
+1 on community is sufficient. Customers are community members, too.